how-to-docs/rds-provisioning.md

```diff
@@ -0,0 +1,137 @@
+# Provisioning an AWS RDS database into the Platform
+
+This requires docker and AWS credentials for hod-dsp-[env] to create the resources and dsp-ci for drone to deploy to the platform. In this example env is _dev_ and the service_name is _myexample_
+
+
+### Workflow
+ - Pull stacks-hod-platform repo
+ - Configure templates in feature branch and push
+ - Create db password and upload secrets
+ - Peer review then merge to master, optionally run script to deploy to prod
+ - Pass access to requester as kubernetes secrets
+
+
+### Source the current configs by git pulling the stacks repo
+
+`$ git clone ssh://git@gitlab.digital.homeoffice.gov.uk:2222/Devops/stacks-hod-platform.git`
+
+### Configure by editing files
+
+* make feature branch and enter the appropriate dev/prod directory.
+  Find templates 
+  
+  `$ ls stacks-hod-platform/stacks/templates/hod-dsp-*`
+
+* RDS: copy a similar existing template based on the db type (postgres/mysql).
+  Change the lines at the top defining the resource name, this example satisfies the request of the name _egteam_ in the prod environment.
+  
+```
+{% set service_name='egteam' %}
+{% set service_env='prod' %}
+{% set dns_name=service_name+'-'+service_env %}
+# the resource name constructed like: {{ service_name }}-{{ service_env }}-rds
+```
+This will be visible in the deployed AWS resource as DBSubnetGroupName": "egteam-prod-rds-..."
+
+  - Change the instance type and database size according to the request
+    `AllocatedStorage: 100`
+    `DBInstanceClass: db.m3.large`
+
```

Comment:
you should also mention setting the database type and version e.g postgres:9.5 and making sure the ports are correct for the database type e.g 3306 for mysql